### PR TITLE
Add dogs

### PR DESCRIPTION
**Pull request by bdoncer**
golden

### CODE REVIEW

File: server/src/main/java/com/demo/Controller.java [hunk 1]
```diff
@@ -38,9 +36,9 @@ public String health() {
     }
 
     @GetMapping("/id")
-    public int generatedId() {
```
Comment (MSpryszynski) [inline server/src/main/java/com/demo/Controller.java:41]: Poor generated word

File: app/public/client.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 document.addEventListener('DOMContentLoaded', () => {
-    const button = document.getElementById('getHealthButton');
-    button.addEventListener('click', async () => {
+    const healthButton = document.getElementById('getHealthButton');
+    healthButton.addEventListener('click', async () => {
```
Comment (anamonium) [inline app/public/client.js:3]: do we still need that button?

File: app/public/client.js [hunk 3]
```diff
@@ -12,11 +12,9 @@ document.addEventListener('DOMContentLoaded', () => {
             console.error('Error fetching health status:', error);
         }
     });
-});
 
-document.addEventListener('DOMContentLoaded', () => {
-    const button = document.getElementById('getIdButton');
-    button.addEventListener('click', async () => {
+    const idButton = document.getElementById('getIdButton');
```
Comment (anamonium) [inline app/public/client.js:16]: same here.

File: encryption/encryptor.go [hunk 4]
```diff
@@ -49,13 +49,15 @@ func main() {
 func eventHandler(ctx context.Context, e *common.TopicEvent) (retry bool, err error) {
 	fmt.Println("Subscriber received:", e.Data)
 	client, _ := dapr.NewClient()
-	encryptDecryptString(client)	
+	if data, ok := e.Data.(string); ok {
+		encryptDecryptString(client, data)
+	} else {
+		fmt.Println("Data is not of type string")
```
Comment (anamonium) [inline encryption/encryptor.go]: shouldn't we return some kind of error?

**Review by MSpryszynski — APPROVED** [review]
Zgiń przepadnij

**Pull request merged**